tools/pythonpkg/scripts/connection_methods.json

```diff
@@ -385,6 +385,15 @@
 				"type": "int"
 			}
 		],
+
+		"kwargs_as_dict": true,
```

Comment:
We don't need `kwargs_as_dict`, if we set this to false we'll just receive a `bool lazy = false` parameter, so we don't have to interrogate and unwrap the kwargs